src/components/button-group/themes/dark/_themes.scss

```diff
@@ -0,0 +1,34 @@
+@use 'sass:map';
+@use 'sass:list';
+@use 'styles/utilities' as *;
+@use 'igniteui-theming/sass/themes/schemas/components/dark/button-group' as *;
+@use '../../../button/themes/button/dark/themes' as *;
+
+$material-flat-btn-size: map.get($material-flat, 'size');
+$bootstrap-flat-btn-size: map.get($bootstrap-flat, 'size');
+$fluent-flat-btn-size: map.get($fluent-flat, 'size');
+$indigo-flat-btn-size: map.get($indigo-flat, 'size');
+$material: map.merge(
+    digest-schema($dark-button-group),
+    (
+        border-radius: border-radius(map.get($dark-button-group, 'border-radius')...),
```

Comment:
You no longer need to resolve the border radius like that. It's already been resolved when digesting the schema.